rules/autoconsent/pubtech.json

```diff
@@ -1,30 +1,30 @@
 {
-    "name": "pubtech",
-    "prehideSelectors": ["#pubtech-cmp"],
-    "detectCmp": [
-      { 
-        "exists": "#pubtech-cmp" 
-      }
-    ],
-    "detectPopup": [
-      { 
-        "visible": "#pubtech-cmp" 
-      }
-    ],
-    "optIn": [
-      { 
-        "click": "#pubtech-cmp #pt-accept-all" 
-      }
-    ],
-    "optOut": [
-      {
-        "click": "#pubtech-cmp #pt-close:first-child" 
-      }
-    ],
-    "test": [
-      {
-        "eval": "document.cookie.includes('euconsent-v2=CPeDpcAPeDpcAFgADAENCaCgAAAAAAAAABCYAAAAAAAA.YAAAAAAAAAAA') && !document.cookie.includes('IDE')"
-      }
-    ]
-  }
-  
\ No newline at end of file
+  "name": "pubtech",
+  "prehideSelectors": ["#pubtech-cmp"],
+  "detectCmp": [
+    { 
+      "exists": "#pubtech-cmp" 
+    }
+  ],
+  "detectPopup": [
+    { 
+      "visible": "#pubtech-cmp #pt-accept-all, #pt-actions"
```

Comment:
did you mean
```
"visible": "#pubtech-cmp #pt-actions,#pubtech-cmp #pt-accept-all",
"check": "any"
```
?

Or maybe just `"visible": "#pubtech-cmp #pt-actions"` is enough?

Reply:
tested it with this option `"visible": "#pubtech-cmp #pt-actions"` and it works